### DISCUSSION TITLE
Welcome to django-sspanel Discussions!

### DISCUSSION THREAD

**Discussion by Ehco1996** [Ehco1996/django-sspanel#455] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sljeff** [comment]
和issue有啥区别

**Reply by Ehco1996** [reply]
这里更像灌水区吧

**Reply by andrew111-oo** [reply]
方便增加vmess协议吗~感谢回复

**Reply by Ehco1996** [reply]
related to https://github.com/Ehco1996/django-sspanel/pull/498
